source/assets/sass/components/_playground.scss

```diff
@@ -0,0 +1,201 @@
+@use 'sass:color';
+@use '../breakpoints';
+@use '../config/color/brand';
+@use '../visual-design/typography';
+
+.playground {
+  --sl-container-max-width: 100vw;
+  display: grid;
+  grid-template:
+    'header' auto
+    'main' 1fr
+    'footer' auto / 100%;
+  min-height: 100vh;
+
+  #main-content {
+    display: grid;
+    grid-template:
+    'editor' 1fr
+    'alert' min-content;
+  }
+}
+
+.sl-r-banner__playground-title {
+  // Use smaller sizes for condensed header
+  --h1-size: var(--sl-font-size--xx-large);
+  --h1-size-lg: var(--sl-playground-heading);
+  margin-bottom: 0;
+  flex: 1;
+}
+
+
+// Playground banner actions area
+.sl-r-banner__playground-actions {
+  flex-direction: row-reverse;
+  gap: var(--sl-gutter);
+}
+
+// Playground banner actions alert message
+.sl-r-banner__playground-alert {
+  padding-left: var(--sl-gutter);
+  padding-right: var(--sl-gutter);
+  position:  var(--playground-alert-position, absolute);
+  right: var(--playground-alert-right, 0);
+  top: -50%;
+
+  &.show {
+    top: auto;
+    right: auto;
+  }
+
+  @media screen and (width >= 1065px) {
+    &.show {
+      --playground-alert-position: relative;
+    }
+  }
+}
+
+#playground-copied-alert {
```

Comment:
@stacyk This id is targeting the same element as the above class, and I think there's some competition between the two. This is causing the exit animation to appear like this-
![image](https://github.com/oddbird/sass-site/assets/167908/61e3b394-f55f-4f28-9476-53577240de77)
It immediately loses height, moves to the corner, and then fades out.